lib/webapp.js

```diff
@@ -1798,7 +1798,29 @@ WebApp.prototype = {
       }
     }
   },
-  
+
+  _installIframePlugin(plugin, urlBase) {
+    if(plugin.webContent && plugin.webContent.framework === 'iframe' && plugin.webContent.destination>'') {
+      const iframeRouterWithMiddleware = [];
+      iframeRouterWithMiddleware.push(this.auth.middleware);
+      iframeRouterWithMiddleware.push(commonMiddleware.setHstsIfSecure());
+      iframeRouterWithMiddleware.push((req,res,next) => {
+          installLog.debug(`Loading iframe app ${plugin.identifier} located at ${plugin.webContent.destination}`);
+          next();
+      });
+      const destination = plugin.webContent.destination;
+      const router = express.Router();
+      router.use((req, res, callback) => {
+        const remoteUrl = zluxUtil.makeRemoteUrl(destination, req);
+        installLog.info(`Loading remote iframe app ${plugin.identifier} located at ${remoteUrl}`);
+        const startingPage = zluxUtil.getRemoteIframeTemplate()(remoteUrl);
```

Comment:
alternatively to a map mentioned in previous file, this _installIframePlugin function could hold the template ahead of time, so router.use already has the result.

Reply:
cool, let me try

Reply:
it just get replace value, after we have remoteURL variable replaced. 
its part of utils module closure